test/common/stats/stats_impl_test.cc

```diff
@@ -478,40 +478,26 @@ TEST(TagProducerTest, CheckConstructor) {
       "No regex specified for tag specifier and no default regex for name: 'test_extractor'");
 }
 
-// Check consistency of internal stat representation
-TEST(RawStatDataTest, Consistency) {
```

Comment:
Note: moved to hot_restart_impl_test.cc, which allocates RawStatData*. stats_impl.cc no longer provides an allocator that allocates RawStatData*.

Reply:
This test -- still in hot_restart_impl_test.cc -- no longer covers truncation. That's covered in a separate new test in thread_local_store_test.cc, which is solely responsible for truncation.

This test's sole purpose is too make sure that the bit-pattern of a shared-memory stat allocation remains unchanged across releases, unless we also update SharedMemory::VERSION.